packages/docs/pages/reference/elements/tabular.mdx

```diff
@@ -8,12 +8,179 @@ import {
 
 # `<tabular>`
 
-Short description of what the `<tabular>` tag does.
+The `<tabular>` tag is the workhorse tag for displaying a collection of cells
+presented in rows and columns.  For complex tables, there is an [online table
+builder](https://www.latex-tables.com/) which can generate PreTeXt output.
+
 
 ## Syntax
 
 <SyntaxDisplay name="tabular" />
 
 ## Examples
 
-    
\ No newline at end of file
+### A tabular example wrapped by an `<activity>` instead of a `<table>`.
+
+```ptx-example
+<FRAGMENT parents="subsection section">
```

Comment:
This example is huge. I think it can be cut down. Also, it appears the tabular is actually wrapped in a `<p>`...is it different for `p > tabular` vs. `activity > p > tabular`?